.gitignore

```diff
@@ -6,6 +6,13 @@ __pycache__/
 # C extensions
 *.so
 
+
```

Comment:
I have been using uv on my mac and I would like to add uv to our gitignore.

This is in line with what github is doing now, see:
https://github.com/github/gitignore/pull/4545

For the same change "upstream".